src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -98,10 +99,22 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('hub_name', hub_name_type, options_list=['--name', '-n'], id_part='name')
         c.argument('etag', options_list=['--etag', '-e'], help='Entity Tag (etag) of the object.')
 
-    for subgroup in ['consumer-group', 'policy', 'job', 'certificate']:
+    for subgroup in ['consumer-group', 'policy', 'job', 'certificate', 'routing-endpoint', 'route']:
         with self.argument_context('iot hub {}'.format(subgroup)) as c:
             c.argument('hub_name', options_list=['--hub-name'])
 
+    with self.argument_context('iot hub routing-endpoint') as c:
```

Comment:
personally I feel it a bit confusing as it is not consistant with the UI. But Paul is working on rewriting the blade for the UI, i think better have him review the command name, help text and parameter names.

Reply:
No issues, ones I got the review feedback I'll update it.